pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +244,32 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		Context("Option set to invalid value", func() {
+			var (
+				err error
+				clientMask    []byte
+				routerIP      net.IP
+				searchDomains []string
+				dhcpOptions   *v1.DHCPOptions
+				options       dhcp4.Options
+			)
+			BeforeEach(func() {
+				options, err = prepareDHCPOptions(clientMask, routerIP, nil, nil, searchDomains, 1500, "myhost", dhcpOptions)
```

Comment:
shouldn't we also have variables for the routes and the dns (I think IP ...) ?

Reply:
Added variables for `dnsIPs, routes and hostname`